client/my-sites/site-settings/spam-filtering-settings.jsx

```diff
@@ -59,25 +64,26 @@ const SpamFilteringSettings = ( {
 		className,
 		header = null;
 
-	if (
-		! shouldShowOfferResetFlow() &&
-		! inTransition &&
-		! ( hasAkismetFeature || hasAntiSpam ) &&
-		! isValidKey
-	) {
+	if ( inTransition ) {
+		return null;
+	}
+
+	if ( ! ( hasAkismetFeature || hasAntiSpamFeature || hasJetpackAntiSpamProduct ) ) {
```

Comment:
Only show the upsell if the site doesn't have any of the following: Akismet as a feature, Jetpack Anti-spam as feature, or Jetpack Anti-spam as a product.